Ooui.Forms/Platform.cs

```diff
@@ -31,6 +32,19 @@ public class Platform : BindableObject, IPlatform, INavigation, IDisposable
         public Platform ()
         {
             _renderer = new PlatformRenderer (this);
+
+            MessagingCenter.Subscribe(this, Page.AlertSignalName, (Page sender, AlertArguments arguments) =>
+            {
```

Comment:
We really should put a check into here to return if there is already a DisplayAlert on the screen. I tried adding a simple bool into the platform to handle the state but this ran into a race condition. If you have a second DisplayAlert that displays based on the result of the first the 2nd display alert will not display due to this race condition